### DISCUSSION TITLE
Why is Microsoft so slow in approving extensions?

### DISCUSSION THREAD

**Discussion by infinitepower18** [microsoft/MicrosoftEdge-Extensions#46] [Q&A]
Does Microsoft really need 7 days to review an extension? I had submitted my extension to all browser extension stores on the same day and Google, Mozilla, Apple and Opera all approved it within a day. The only store where it hasn't been approved yet is the Edge extension store due to their long review times.

This is the second time I've submitted my extension for review, since my first one required more info in the review notes (I'll admit that was my fault for not giving enough info). The first one took 7 days to review, and I'm still waiting for the review team to take a look at my second one. Disappointed with the process so far.

**Comment by Jaifroid** [comment]
Still a serious issue. The Edge Extension store is the slowest by a country mile of all the other Stores (Chrome, Mozilla, WinGet, and even the non-extension Microsoft Store). I'm sure it has a lot fewer submissions to deal with than Google. I can't justify encouraging users to use the Edge Store version of my app, when they will get updates a lot faster if they simply go and install it from the Chrome Store. If Microsoft wants to compete seriously with Chrome, this is a funny way of going about it. MS should be nimbler, faster, offer tangible benefits over Chrome if it wants users to switch to Edge.

**Comment by Gurjeet-msft** [comment]
@Jaifroid and @infinitepower18 Hello, 

The review times for Edge extension submissions has improved in the recent times. Please provide us more details of your extensions and the submission you are referring to @[ext_dev_support@microsoft.com]

**Reply by infinitepower18** [reply]
It still took 5 days the last time I submitted my Clone in VS Code extension update back in July. Others approve within 24 hours. Sorry but 5 days is way too long to get an extension approved.

**Reply by akopachov** [reply]
Same for [my extension](https://github.com/akopachov/sveltab). It takes less than a day to publish update to Chrome\Firefox and around 5 days to publish same update to Edge store. Why so slow?

**Reply by YanzhongSu** [reply]
It took exactly 18 hours to get my [extension](https://chromewebstore.google.com/detail/cbkiaocamdmihicmjkpoefhlgiioipmb?utm_source=item-share-cb) published on Chrome, but 6 working days on Edge. I always tell my user to use Chrome version instead.

**Comment by aiktb** [comment]
> Same for [my extension](https://github.com/akopachov/sveltab). It takes less than a day to publish update to Chrome\Firefox and around 5 days to publish same update to Edge store. Why so slow?

For me, 5 days is a luxury. It has been 14 business days without any response, and I can no longer bear it.😅

**Reply by prithamkasiMSFT** [reply]
Hi @aiktb 

Apologies for the inconvenience caused to you, could you please share the details @ [ext_dev_support@microsoft.com] and we will try to resolve it as soon as possible.

**Comment by Jaifroid** [comment]
See also #131.

**Comment by infinitepower18** [comment]
@ManikanthMSFT there has been no improvement to the review time. The portal still says 7 business days needed. Why did you close this discussion as resolved?

**Reply by ManikanthMSFT** [reply]
@infinitepower18, could you please provide us your seller ID and extension details so that we can verify and assist you further.

**Reply by infinitepower18** [reply]
@ManikanthMSFT 

Extension: https://microsoftedge.microsoft.com/addons/detail/idolkdgdllilclecodkncimdbmmclmje
Seller ID: 23259670

**Reply by ManikanthMSFT** [reply]
Hi @infinitepower18, your extension is currently in the review process. We understand the importance of timely updates and recommend allowing up to three business days for this process to complete.

If the review is not finalized within this timeframe, we will escalate the matter to ensure it is addressed promptly.

We appreciate your understanding

**Comment by emiraydin** [comment]
To the employee claiming review times have improved: How exactly? I waited 1.5 months to get verified into Partner Center to be able to submit an extension. Then I waited 10 days to get a review at the end of which some reviewer tried it for 30 seconds and said it does not work and rejected it. I have been in Chrome and Firefox app stores in well over a month now at version 1.2 whereas I still could not get version 1.0 into Edge App Store. All subsequent versions took hours to get approved in both Chrome and Firefox app stores. Microsoft developer experience is a complete shitshow and I will be forced to pull my app from the Edge store if it goes on like this and ask users to use the Chrome/Firefox extensions.

**Comment by 92jackson** [comment]
Agreed, this is a joke! Updates I push to FireFox are approved (quite literally) within 10 minutes of posting– with Edge, I'm waiting at least 5 days, by which point I'm ready to push another update.

Really not sure why this has been marked resolved. You really need to do better Microsoft.

**Comment by hsinyu-chen** [comment]
do you even want devs submit extensions on your store front ? 5 day is way to long , chrome usually only take 1-2 hours to get update approved
